### ISSUE TITLE
Please create hardware version with ESP32-S3 or at least PSRAM :) + USB serial pins connected 

### ISSUE THREAD

**Issue by clowrey** [GeekMagicClock/smalltvpro#44]
A version with the PSRAM would be amazing for ESPhome + Home Assistant uses :) ESPhome LVGL component uses allot of ram so cannot easily run without PSRAM. If you can also use the S3 module it has built in USB transceiver so could have USB programming and debugging for DIY uses would be really nice :) maybe then you could release some simple open source firmware, or just let the community create it which they will have no problem doing, it has already been done on current versions.

If you wanted a simpler method with NO PCB changes just using different ESP32 module there is now the new version ESP32-WROOM-32E-N8R2 which includes 2MB PSRAM in the same footprint etc, price is very similar too.. So could sell a version with that for a few more $$ and I would buy it. 

https://community.home-assistant.io/t/installing-esphome-on-geekmagic-smart-weather-clock-smalltv-pro/618029/34
https://www.mouser.com/ProductDetail/Espressif-Systems/ESP32-WROOM-32E-N8R2?qs=By6Nw2ByBD0mvGJSS3Ohlw%3D%3D

**Comment by GeekMagicClock** [comment]
Hi, thanks for the suggestion.
Yes, I'm gona to make it 16MB flash + 8MB PSRAM version, so people would make something without worry about poor little size of rom/ram. 

The basic development of the project will be open source, there's many protential for you guys to do with it, expecting your creations.

**Comment by clowrey** [comment]
I can confirm that if you replace the ESP32 module with the ESP32-WROOM-32E-N8R2 version with 2MB PSRAM it will work with LVGL + ESPhome :) It is allot of work to replace the module though so I would recommend waiting for the DIY version.. 
My code is at the link below:
https://github.com/clowrey/esphome-geekmagic-small-TV-pro/tree/main
![geek_psram4](https://github.com/GeekMagicClock/smalltv-pro/assets/6935928/aea060b7-3a1b-433e-9800-92bd04241a9c)

**Comment by clowrey** [comment]
@GeekMagicClock if you can make one with a touchscreen that would be really cool too ;) not sure how much that would add to the cost.. but would make it much more versatile. ESPhome already supports this common one: https://esphome.io/components/touchscreen/cst816.html

**Comment by GeekMagicClock** [comment]
> @GeekMagicClock if you can make one with a touchscreen that would be really cool too ;) not sure how much that would add to the cost.. but would make it much more versatile. ESPhome already supports this common one: https://esphome.io/components/touchscreen/cst816.html

Yeah, touch version may will be future vesions. I have already made some samples without touch, should be soon to ready.

**Comment by SirSharanChakradhar** [comment]
@GeekMagicClock Thank you! Is the new version ready yet? When should we expect? Also, There is no where in the Internet about the hardware details at all. What are the hardware differences between Small tv, Pro and Ultra?

**Comment by GeekMagicClock** [comment]
https://www.aliexpress.com/item/3256807368227919.html
Hello, after a long time review, it is on stock now, thank you for the interest.

**Comment by clowrey** [comment]
Thanks! It looks good! I just wish the price was a little lower..

**Comment by GeekMagicClock** [comment]
> Thanks! It looks good! I just wish the price was a little lower.. I still bought one but I would buy 5 or 10pcs if price was lower like $20 similar to other S3 dev boards with LCD. I think marketing it as an ESPhome display for use with Home Assistant will get LOTS of users interested in it over time. Home Assistant is one of the largest open source projects on Github.

OK, I tried setting up the discount, but maybe it looks like it doesn't work in some countries, I'm checking the issue, you can check the link later.

**Comment by adosix** [comment]
Hi, 
I saw the product and bought it right away but I am not sure if my PRO version can be used with ESPhome. Can you give me a hint how to find out?
[https://www.aliexpress.com/item/1005005132140010.html?spm=a2g0o.order_list.order_list_main.17.1ab01802ugrvzU#nav-specification](https://www.aliexpress.com/item/1005005132140010.html?spm=a2g0o.order_list.order_list_main.17.1ab01802ugrvzU#nav-specification)
